src/MiniCover/Commands/Options/IncludeFilesPatternOption.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Extensions.CommandLineUtils;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MiniCover.Commands.Options
+{
+    public abstract class IncludeFilesPatternOption : MiniCoverOption<IEnumerable<string>>
```

Comment:
Maybe rename to FilesPatternOption. Just because Exclude also inherits from Include.